Tests/test_imagecms.py

```diff
@@ -174,19 +174,24 @@ def test_exceptions():
     psRGB = ImageCms.createProfile("sRGB")
     pLab = ImageCms.createProfile("LAB")
     t = ImageCms.buildTransform(pLab, psRGB, "LAB", "RGB")
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="mode mismatch"):
         t.apply_in_place(hopper("RGBA"))
 
     # the procedural pyCMS API uses PyCMSError for all sorts of errors
     with hopper() as im:
-        with pytest.raises(ImageCms.PyCMSError):
+        with pytest.raises(ImageCms.PyCMSError, match="cannot open profile file"):
             ImageCms.profileToProfile(im, "foo", "bar")
-    with pytest.raises(ImageCms.PyCMSError):
+
+    with pytest.raises(ImageCms.PyCMSError, match="cannot open profile file"):
         ImageCms.buildTransform("foo", "bar", "RGB", "RGB")
-    with pytest.raises(ImageCms.PyCMSError):
+
+    with pytest.raises(ImageCms.PyCMSError, match="Invalid type for Profile"):
         ImageCms.getProfileName(None)
     skip_missing()
-    with pytest.raises(ImageCms.PyCMSError):
+
+    # Windows, Python <= 3.9: "an integer is required (got type NoneType)"
+    # Others: "'NoneType' object cannot be interpreted as an integer"
```

Comment:
```suggestion
    # Python <= 3.9: "an integer is required (got type NoneType)"
    # Python > 3.9: "'NoneType' object cannot be interpreted as an integer"
```
See macOS [3.8](https://github.com/radarhere/Pillow/runs/6953537130?check_suite_focus=true#step:8:1590), [3.9](https://github.com/radarhere/Pillow/runs/6953537092?check_suite_focus=true#step:8:1590) and [3.10](https://github.com/radarhere/Pillow/runs/6953537060?check_suite_focus=true#step:8:1589), Ubuntu [3.8](https://github.com/radarhere/Pillow/runs/6953537303?check_suite_focus=true#step:8:1591), [3.9](https://github.com/radarhere/Pillow/runs/6953537279?check_suite_focus=true#step:8:1591) and [3.10](https://github.com/radarhere/Pillow/runs/6953537256?check_suite_focus=true#step:8:1590)